pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -632,17 +655,20 @@ def _distribution_new_customers(
             s = pm.HalfFlat("s")
             beta = pm.HalfFlat("beta")
 
-            # This is the shape if using fit_method="map"
-            if self.fit_result.dims == {"chain": 1, "draw": 1}:
-                shape_kwargs = {"shape": 1000}
-            else:
-                shape_kwargs = {}
-
             pm.Gamma(
                 "population_purchase_rate", alpha=r, beta=1 / alpha, **shape_kwargs
             )
             pm.Gamma("population_dropout", alpha=s, beta=1 / beta, **shape_kwargs)
 
+            ParetoNBD(
+                name="customer_population",
+                r=r,
+                alpha=alpha,
+                s=s,
+                beta=beta,
+                T=T,
+            )
```

Comment:
Hey @ColtAllen we got this merged already but perhaps we can improve in a future PR. I am thinking of splitting the recency and number of purchases into two separate variables instead of having the current vector (n, 2)? Somethingl like 

```
pnbd = ParetoNBD(
    name="customer_behavior",
    r=r,
    alpha=alpha,
    s=s,
    beta=beta,
    T=T,
)

pm.Deterministic("customer_recency", pnbd[..., 0])
pm.Deterministic("customer_num_purchases", pnbd[..., 1])
```

Would also be good to add dims `customer_id` to these variables. WDYT?

Reply:
Opened issue in #417